pkg/testdata/models/apis/ec2/0000-00-00/generator.yaml

```diff
@@ -68,8 +68,11 @@ operations:
     output_wrapper_field_path: VpcEndpoint
 
 resources:
-  DHCPOptions:
-
+  DhcpOptions:
+    fields:
+      DHCPConfigurations.Values:
+        set:
+          - from: AttributeValue.Value
```

Comment:
Is it possible to add a test case for multiple nested fields? something similar to `A.B.C`

Reply:
Did you mean multiple nested fields for the *Resource* or *Shape*? Note, this is a nested example for **both**